docs/casestudies/adopters.md

```diff
@@ -21,3 +21,4 @@ All organisations are sorted alphabetically.
 | Trip.com                  | https://www.trip.com/       | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada            | TBD                                                                           |
 | VIPKID                    | https://www.vipkid.com/     | Use Karmada to build multi-cluster platform with heterogeneous resources and clusters   | [Building a PaaS Platform with Karmada to Run Containers --VIPKID](vipkid.md) |
 | Youzan                    | https://www.youzan.com/     | Manage both Stateless and Stateful on-line workloads across multiple clusters           | TBD                                                                           |
+| Vivo                      | https://www.vivo.com/       | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada            | TBD                                                                           |
```

Comment:
Given `All organisations are sorted alphabetically.`, I think you can put the new item right after `VIPKID` and before `Youzan`.

Reply:
done